Tests/DefaultXpringClientTest.swift

```diff
@@ -418,6 +418,66 @@ final class DefaultXpringClientTest: XCTestCase {
     }
   }
 
+  // MARK: - TransactionHistory
```

Comment:
What does this `MARK` mean?

Reply:
keefer explains this here: https://github.com/xpring-eng/XpringKit/pull/116#discussion_r387110813